packages/web-frontend/src/screens/exporter/RootScreen/index.js

```diff
@@ -13,31 +13,39 @@
  * displays the correct chart.
  */
 import React, { PureComponent } from 'react';
-import { connect } from 'react-redux';
 import queryString from 'query-string';
 import moment from 'moment-timezone';
 
 import { ChartWrapper, getIsMatrix, MatrixWrapper } from '../../../components/View';
 import { request } from '../../../utils';
 import { DARK_BLUE, WHITE } from '../../../styles';
-import { selectOrgUnit } from '../../../selectors';
+import { getInitialLocationComponents, URL_COMPONENTS } from '../../../historyNavigation';
 
-// TODO: Export functionality to come in future PR.
-/*
-const initialLocation = getInitialLocation();
-*/
 const {
-  organisationUnitCode,
-  organisationUnitName,
-  dashboardId,
-  reportId,
-  timeZone,
-  startDate,
-  endDate,
-  disasterStartDate,
-  disasterEndDate,
-  projectCode,
-} = {}; //decodeLocation(initialLocation);
+  PROJECT,
+  ORG_UNIT,
+  DASHBOARD,
+  REPORT,
+  TIMEZONE,
+  START_DATE,
+  END_DATE,
+  DISASTER_START_DATE,
+  DISASTER_END_DATE,
+  ORG_UNIT_NAME,
+} = URL_COMPONENTS;
```

Comment:
This is really verbose and unnecessary... Is there a cleaner syntax to import all of these constants, or another way to approach this?

Reply:
I think not destructuring the constants here and using them directly will make this clean enough:

```js
const {
  [URL_COMPONENTS.ORG_UNIT]: organisationUnitCode,
  [URL_COMPONENTS.ORG_UNIT_NAME]: organisationUnitName,
  // etc
} = getInitialLocationComponents();
```

I have generally come to favour not destructuring "enum" constants, since `ENUM_NAME.KEY` highlights the `key` is part of an enum, plus `ENUM_NAME` gives the appropriate context (eg a made up example: `URL_COMPONENTS.FACILITY`, `ENTITY_TYPES.FACILITY` )

Reply:
I like that! Still pretty fat lines for something which conceptually is very simple, but never mind.